IChat/IChat/Messanger/MainViewController.swift

```diff
@@ -0,0 +1,50 @@
+import UIKit
+
+class MainViewController: UITabBarController {
+
+    private var message: String?
+    private var contact: String?
+    private var mobile: String?
+    
+    init(message: String? = "Dummy", contact: String? = "Dummy", mobile: String? = "Dummy") {
+        self.message = message
+        self.contact = contact
+        self.mobile = mobile
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tabBar.itemPositioning = .fill
+        setupView()
+    }
+
+    func setupView() {
+        let chatVC = createViewController(with: ChatsViewController(message: message), title: "Chats", imageName: "chats")
```

Comment:
Avoid using non-standard acronym like `VC for view controller` for variable/function/class names. Instead try to provide some meaningful complete name, may be based on their functionality/usage.


```suggestion
        let chatViewController = createViewController(with: ChatsViewController(message: message), title: "Chats", imageName: "chats")
```